src/Workspaces/Core/Portable/Editing/ImportAdderService.cs

```diff
@@ -121,12 +121,14 @@ private static SyntaxNode GenerateNamespaceImportDeclaration(INamespaceSymbol na
             var model = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
 
             var nodesWithExplicitNamespaces = syntaxNodes
-                .Select(n => (syntaxnode: n, namespaceSymbol: GetExplicitNamespaceSymbol(n, model)))
-                .Where(x => x.namespaceSymbol != null);
```

Comment:
Why was this removed? haven't we just moved to null check into the foreach loop now?

Reply:
Yes. Moving the null check into the foreach is necessary for the compiler to realize that 'namespaceSymbol' is not null. Before the compiler just assumed 'namespaceSymbol' was not null. See https://github.com/dotnet/roslyn/pull/52722/files#r782643742